### PR TITLE
skip network tests when kube-calico ds doesn't exist

### PR DESCRIPTION
**Pull request by abhinavdahiya**
https://github.com/kubernetes-incubator/bootkube/issues/605#issuecomment-310480674

### CODE REVIEW

File: e2e/network_test.go [hunk 1]
```diff
@@ -15,6 +16,12 @@ import (
 )
 
 func TestNetwork(t *testing.T) {
+	// check if kube-calico daemonset exists
+	// if absent skip this test
+	if _, err := client.ExtensionsV1beta1().DaemonSets("kube-system").Get("kube-calico", metav1.GetOptions{}); err != nil && apierrors.IsNotFound(err) {
```
Comment (aaronlevy) [inline e2e/network_test.go]: I'm okay with this if we think of the name being stable. If not, is there a label query we could rely on instead?

Comment (aaronlevy) [inline e2e/network_test.go]: I guess name is up to us, so probably a non-issue

**Comment by aaronlevy** [comment]
Post-lgtm nit: seems a little weird to ignore the error we return (or should add comment or don't assign it at all).

**Comment by aaronlevy** [comment]
post-lgtm-lgtm

**Comment by abhinavdahiya** [comment]
Fixed in https://github.com/kubernetes-incubator/bootkube/pull/639

**Pull request closed without merging**